Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Would add more info here that when used as a prior it's often used for parameters like in a regression. Or maybe add that as an example below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Added</p>